src/components/Table/Table.vue

```diff
@@ -8,7 +11,15 @@
                     :header="header"
                     :name="`header-${header.key}`"
                 >
-                    <th :key="header.key">
+                    <th
+                        :key="header.key"
+                        :class="{
+                            sortable: header.sortable,
+                            asc: sorted[header.key] === 'asc',
+                            desc: sorted[header.key] === 'desc',
```

Comment:
I think you should make the classes a little more specific than this to prevent conflicts with other class names.
```suggestion
                            'table-heading-sortable': header.sortable,
                            'table-heading-sortable-asc': sorted[header.key] === 'asc',
                            'table-heading-sortable-desc': sorted[header.key] === 'desc',
```